src/Microsoft.PowerShell.Commands.Utility/commands/utility/XmlCommands.cs

```diff
@@ -359,6 +360,255 @@ protected override void StopProcessing()
         }
     }
 
+    #region ConvertTo-Clixml
+    /// <summary>
+    /// Implements ConvertTo-Clixml command.
+    /// </summary>
+    [Cmdlet(VerbsData.ConvertTo, "Clixml", SupportsPaging = true, DefaultParameterSetName = "ByPath", HelpUri = "")]
+    public sealed class ConvertToClicmlCommand : PSCmdlet, IDisposable
+    {
+        #region Command Line Parameters
+
+        /// <summary>
+        /// Gets or sets input object to be converted.
+        /// </summary>
+        [Parameter(ValueFromPipeline = true, Mandatory = true)]
+        [AllowNull]
+        public PSObject InputObject { get; set; }
+
+        /// <summary>
+        /// Gets or sets depth of serialization.
+        /// </summary>
+        [Parameter]
+        [ValidateRange(1, int.MaxValue)]
+        public int Depth { get; set; } = 0;
+
+        /// <summary>
+        /// Gets or sets encoding.
+        /// </summary>
+        [Parameter]
+        [ArgumentToEncodingTransformationAttribute]
+        [ArgumentEncodingCompletionsAttribute]
+        [ValidateNotNullOrEmpty]
+        public Encoding Encoding { get; set; } = ClrFacade.GetDefaultEncoding();
+
+        #endregion Command Line Parameters
+
+        #region Overrides
+
+        /// <summary>
+        /// Begin processing.
+        /// </summary>
+        protected override void
+        BeginProcessing()
```

Comment:
```suggestion
        protected override void BeginProcessing()
```
Please fix below too.